rasa/train.py

```diff
@@ -697,19 +697,17 @@ def train_nlu(
         otherwise the path to the directory with the trained model files.
 
     """
-    return rasa.utils.common.run_in_loop(
-        _train_nlu_async(
-            config,
-            nlu_data,
-            output,
-            train_path,
-            fixed_model_name,
-            persist_nlu_training_data,
-            additional_arguments,
-            domain=domain,
-            model_to_finetune=model_to_finetune,
-            finetuning_epoch_fraction=finetuning_epoch_fraction,
-        )
+    return await _train_nlu_async(
```

Comment:
why was this change required?

Reply:
Because I had errors of type `RuntimeError: this event loop is already running.`. 

I just checked it again and for example for these tests  : 
```
FAILED tests/test_train.py::test_train_nlu_temp_files - RuntimeError: this event loop is already running.
FAILED tests/test_train.py::test_train_nlu_wrong_format_error_message - RuntimeError: this event loop is alre...
FAILED tests/test_train.py::test_train_nlu_with_responses_no_domain_warns - RuntimeError: this event loop is ...
FAILED tests/test_train.py::test_train_nlu_with_responses_and_domain_no_warns - RuntimeError: this event loop...
FAILED tests/test_train.py::test_train_nlu_no_nlu_file_error_message - RuntimeError: this event loop is alrea...
FAILED tests/test_train.py::test_train_nlu_autoconfig - RuntimeError: this event loop is already running.
```
I get the above errors when I have the `run_in_loop`. When I change to `await` the tests pass.

Reply:
Mhm, interesting 🤔  Why did this error happen? What's the stack trace? Could it be that [this](https://github.com/RasaHQ/rasa/blob/a7387f445604be5d123b776a2f4d1521990b8453/rasa/nlu/test.py#L1900) should rather call the `async` version of `train_nlu` (which is `_train_nlu_async`)?

Reply:
@wochinge Hey Tobias! 
Here are the error messages [trace.txt](https://github.com/RasaHQ/rasa/files/5768791/trace.txt)
I can check and try to call `_train_nlu_async`

Reply:
I double checked for `test_train_nlu_temp_files` and if you don't make the test function `async`, you can use `rasa.utils.common.run_in_loop` as before. 

The issue is that when you define a test function via

```python
async def test_something():
   pass
```
it will start a event loop. When your test code starts another event loop via `rasa.utils.common.run_in_loop` you get the error above. As you don't need the `async` for anything in this test, you can remove it safely.